arch/arm/soc/st_stm32/stm32f1/Kconfig.series

```diff
@@ -14,5 +14,6 @@ config SOC_SERIES_STM32F1X
 	select HAS_STM32CUBE
 	select CPU_HAS_SYSTICK
 	select CLOCK_CONTROL_STM32_CUBE if CLOCK_CONTROL
+	select I2C_STM32_V1 if I2C
```

Comment:
I think i moved this out of here take a look at:

commit 8e8d49a6202a472513354b1db27d2b4ffd23faa5
Author: Kumar Gala <kumar.gala@linaro.org>
Date:   Wed Jul 5 11:04:06 2017 -0500

    i2c: stm32: Cleanup how we enable the specific I2C driver